Makefile

```diff
@@ -338,6 +338,10 @@ test.unit:
 test.unit.pretty:
 	@$(MAKE) GOTESTSUM_FORMAT=pkgname _test.unit
 
+.PHONY: test.golden.update
+test.golden.update:
+	@UPDATE_GOLDEN=true go test -v -run TestParser_GoldenTests ./internal/dataplane/parser
```

Comment:
I'm no opposing the usage of env here but I wanted to call out that one could also use a CLI flag only available in tests, e.g.

```

var udpate = flag.Bool("update", false, "update golden files")

func TestX(t *testing.T) {
	t.Logf("updating golden files: %v", *udpate)
}
```

and use it like so:


```
go test -v -count 1 -run TestX . -args -update
```